pkg/apis/config/feature_flags.go

```diff
@@ -31,12 +31,14 @@ const (
 	disableCredsInitKey                     = "disable-creds-init"
 	runningInEnvWithInjectedSidecarsKey     = "running-in-environment-with-injected-sidecars"
 	requireGitSSHSecretKnownHostsKey        = "require-git-ssh-secret-known-hosts" // nolint: gosec
+	enableTektonOCIBundles                  = "enable-tekton-oci-bundles"
```

Comment:
It would be nice to update the feature_flags_test.go and related test files too

Reply:
fixed